tests/juice-shop-negative-login.spec.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+const assert = require('assert');
+const loginPage = require('../pages/JuiceShop/loginPage');
+
+describe('Negative Juice shop login', () => {
+    it('Should not login to Juice shop', async () => {
+        await loginPage.navigateToLoginPage();
+        await loginPage.closeWelcomeDialog();
+        const userData = { emailField: 'ted@test.co', passwordField: '1234' };
```

Comment:
let me put my 2 cents here. This syntax breaks 3A principle (Arrange, Act and Assert (AAA)). It suggests that you should divide your test method into three sections: arrange, act and assert. Each one of them only responsible for the part in which they are named after.
So yet again, great example of how to use the latest language features to minimize code, but to my mind, it is not suitable in this particular case

More readings on this topic:
http://wiki.c2.com/?ArrangeActAssert
https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80